src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -137,15 +143,39 @@ public CommandBase moveTicks(double ticks, double speed) {
                 CommandScheduler.getInstance().requiring(this).cancel();
             }
         }, () -> {
-            return Math.abs(liftEncoder.getDistance()) >= ticks;
+            return Math.abs(liftEncoder.getDistance()) >= Math.abs(0.10);
```

Comment:
Is the `abs` necessary? It's a constant... Also why isn't it a *named* constant?

Reply:
pretty sure we don't quite need it. resolvification complete.